doc/source/appendices/tutorial.rst

```diff
@@ -23,199 +23,126 @@ Notes:
-
-Build the code::
-
-  ./icepack.build
-
-The build script basically runs gmake under the covers, but there are a number of other tasks that are handled by the script to make the build more robust.  If the build is successful you will see the message “COMPILE SUCCESSFUL” at the bottom of the screen. You can also check the README.case file to check the status.
-
-Submit the job. The submit script just submits the run scripts. Look at both **icepack.run** and **icepack.submit** files to see more details. The out-of-the-box run has default settings for the physics and other options. You can have a look at **icepack_in** and **icepack.settings** to review those settings. Then::
-
-  ./icepack.submit
-
-If the run is successful, you will see the message “ICEPACK COMPLETED SUCCESSFULLY” in the icepack run log file. Note that this job runs quickly - you are running a column model with four grid cells!
-
-Look at the output!  Go to the ICE_RUNDIR where output was created. A successful model integration will create ice_diag.* files and a file in the “restart” directory called “iced.2016-01-01-00000”. The Icepack documentation has more information about :ref:`history`.
-
-Follow the documentation to create some plots of the output using the tools provided with Icepack (:ref:`testplotting`). The conda icepack environment must be activated, if it isn’t already::
- 
-  cd ~/icepack-dirs/Icepack/configuration/scripts/tests/
-  conda activate icepack
-  ./timeseries.csh ~/icepack-dirs/runs/icepack_test0/ice_diag.full_ITD
-
-Note that you can run the plotting script on any of the four ice_diag.* files.  The .png files are created in the ICE_RUNDIR directory. Open the files::
-
-  cd ~/icepack-dirs/runs/icepack_test0/
-  open <figurename>.png
-
-Or use your file browser to navigate to the directory and double click on the images.
-
-Questions to think about while looking at the output.
-
-* What time period does an out-of-the-box run cover? 
-* What are the differences between the full_ITD plots and the icefree plots (or any other combination of the ice_diag.* output files)? Which fields are the same? Which are different? Why would this be?
-* What happens to ice area and ice thickness around October 1, 2015? Why do you see this signal?
-* How does your output compare to the sample output provided for this release? (hint: see the wiki!)
-
-Take a step back and think about all the directories and files you have created. The Icepack “sandbox” was cloned from GitHub and has the actual Icepack code.
-
-* There is a particular case directory for building and launching the code, and some output (e.g. job log) are copied.
-* There is a particular run directory for each case. This is where the model is run and big files are found.
-
-
-Set Up a Longer Run
```

Comment:
I think setting up a longer run and modifying a physics option in namelist or a code parameter are useful parts of the tutorial separate from adding a tracer.  I recommend making section 6.2.3 be something like "Modify the configuration or code" and section 6.2.4 "Add a new tracer"

Reply:
I reorganized this as suggested.